src/tx.sw

```diff
@@ -217,3 +217,12 @@ pub fn tx_output_type(ptr: u32) -> u8 {
         r1: u8
     }
 }
+
+/// Get the amount of coins to send for an output given a pointer to the output.
```

Comment:
I guess not all outputs have an `amount` field. Maybe I should clarify the comment to say that this method is only meaningful for
`OutputCoin`
`OutputWithdrawal`
`OutputChange`
`OutputVariable`

Reply:
Yes, please add a comment to that effect: https://github.com/FuelLabs/sway-lib-std/blob/ea88b07066685fc5e642341efff06709fd06c5df/src/tx.sw#L182-L183